pkg/phases/photon.go

```diff
@@ -0,0 +1,28 @@
+package phases
+
+import (
+	"github.com/flanksource/konfigadm/pkg/types"
+	"github.com/flanksource/konfigadm/pkg/utils"
+)
+
+var Photon = photon{}
+
+type photon struct {
+}
+
+func (p photon) GetPackageManager() types.PackageManager {
+	return YumPackageManager{}
```

Comment:
I think you will need to implement a TDNF package manager: https://vmware.github.io/photon/assets/files/html/3.0/photon_admin/tdnf.html

Reply:
Okay, I'll implement TDNF package manager